packages/jest-runner/src/runTest.ts

```diff
@@ -206,13 +206,24 @@ async function runTestInternal(
     },
   };
 
+  const isBrowserEnv = typeof environment.global?.document !== 'undefined';
+
   // For tests
-  runtime
-    .requireInternalModule<typeof import('source-map-support')>(
-      require.resolve('source-map-support'),
-      'source-map-support',
-    )
-    .install(sourcemapOptions);
+  if (isBrowserEnv) {
+    runtime.requireInternalModule(
```

Comment:
for some reason this is _horribly_ slow:

![image](https://user-images.githubusercontent.com/1404810/111865842-a6ec2f80-8969-11eb-93fb-04ca90f5d32b.png)

If I comment it out:

![image](https://user-images.githubusercontent.com/1404810/111865849-b23f5b00-8969-11eb-8d3b-3fccc3dccc4c.png)

Reply:
@LinusU any ideas about what's happening here? I'm having some issues reproducing this using plain `JSDOM`, so I'm not really sure where to begin debugging this...